portfolio/src/main/webapp/index.html

```diff
@@ -8,100 +8,144 @@
+                            successfully and drew a 2D floor plan on a wood board (3m x 6m).
                         </p></li>
                     </ul>    
-                </li>
+                </li><br>
             </ul>
         </div>
 
         <div id="display-container">
-            <h2>Activities</h2>
+            <h2>Activities</h2><br>
             <ul>
-                <li><p>Team Member</p>
+                <li><p id="bold">Team Member</p>
                     <p id="title">Poly Programming Team</p>
                     <p>Engage in team activities and contribute in weekly practice for annual ACM-ICPC regional contest.</p>
-                </li>
-                <li><p>Preception Sub-Team Member</p>
+                </li><br>
+                <li><p id="bold">Preception Sub-Team Member</p>
                     <p id="title">NYU Self Drive</p>
                     <p>
-                      Work with computer vision (PyTorch) to develop machine learning algorithms for an autonomous self-driving 
-                      mini car. Attend weekly meetings and lectures.
+                        Work with computer vision (PyTorch) to develop machine learning algorithms for an autonomous self-driving
+                        mini car. Attend weekly meetings and lectures.
                     </p>
+                </li><br>
             </ul>
         </div>
 
         <div id="display-container">
-            <h2>Education</h2>
-            <ul>
-                <li><p>New York University</p>
+            <h2>Education</h2><br>
+            <ul><li><ul>
+                <li><p id="bold">New York University</p>
                 <p>Bachler of Science - BS, Computer Science</p>
-                <p>Minors in Business Studies, Finance</p>
-                </li>
-                <li><p>Hamden Hall Country Day School</p></li>
-                <li><p>Yale Young Global Scholars</p></li>
-                <li><p>Engineering Summer Academy at Penn</p></li>
-            </ul>
+                <p>Minors in Business Studies, Finance</p></li>
+                <li><p id="bold">Hamden Hall Country Day School</p></li>
+                <li><p id="bold">Yale Young Global Scholars</p></li>
+                <li><p id="bold">Engineering Summer Academy at Penn</p></li>
+            </ul></li></ul>
         </div>
+        <hr>
 
-        <em>Greeting for you:</em>
-        <button onclick="addRandomGreeting()">Hello</button>
-        <div id="greeting-container"></div>
+        <h3>Leave a comment!</h3>
+        <script>
+            checkLoginStatus();
+        </script>
+
+        <div id="login"></div>
+        
+        <div id="comment">
+            <form action="/comment" method="POST">
+                <label for="user-comment">Type in any text:</label><br>
+                <input type="text" name="user-comment" id="user-comment" value="turnitup"><br>
+                <br><input type="submit" value="readytogo">
```

Comment:
Same for the name. 'readytogo' is not easy to read.

Reply:
All names of the buttons are changed! ;)